index.html

```diff
@@ -1,111 +1,173 @@
+    <nav class="navigation">
+      <ul class="navigation-list">
+        <li class="navigation item"><a href="" class="nav-link">Home</a></li>
+        <li class="navigation item"><a href="" class="nav-link">Blog</a></li>
+        <li class="navigation-item"><a href="" class="nav-link">Catalog</a></li>
+        <li class="navigation-item"><a href="" class="nav-link">Contact Us</a></li>
+      </ul>
+    </nav>
+    <button class="search-bar">Search</button>
+    <a href="" class="discord">Discord</a>
+    <button class="connect">Connect</button>
+  </header>
+  <main class="content">
+    <section class="headline">
+      <h1>With <strong>Binabox</strong> Explore NFT collection</h1>
+      <p>We are the best way to check the rarity of NFT collection</p>
+      <ul class="achievements">
+       <li class="achievement-info">2240+</li>
+       <li class="achievement-info">Total item</li>
+       <li class="achievement-info">1000+</li>
+       <li class="achievement-info">Profile whitelisted</li>
+      </ul>
+    </section>
+    <section class="catalog">
+      <h2>Catalog</h2>
+      <a href="">See all</a>
+      <button>Right</button>
+      <button>Left</button>
+      <div class="catalog-item">
+        <img src="" alt="" class="catalog-image">
+        <p class="catalog-price">$199</p>
+        <p class="catalog-describtion">Name product</p>
+      </div>
+      <div class="catalog-item">
+        <img src="" alt="" class="catalog-image">
+        <p class="catalog-price">$199</p>
+        <p class="catalog-describtion">Name Product</p>
+      </div>
+      <div class="catalog-item">
+        <img src="" alt="" class="catalog-image">
+        <p class="catalog-price">$199</p>
+        <p class="catalog-describtion">Name Product</p>
+      </div>
+      <div class="catalog-item">
+        <img src="" alt="" class="catalog-image">
+        <p class="catalog-price">$199</p>
+        <p class="catalog-describtion">Name Product</p>
+      </div>
+    </section>
+    <section class="our-blog">
+      <h2>Our blog</h2>
+      <article class="sonic-live">
+        <img src="" alt="big-face" class="big-face-image">
+        <div class="p">Tony Nguyen</div>
+        <p class="time">20 Jun 2022</p>
+        <h3>Sonic live</h3>
+        <p>One September 22, 2021 A volutpat nulla eros, nulla ultrices nec nulla imperdiet...</p>
+        <a href="" class="read-more">Read more</a>
+      </article>
+      <a href="" class="read-more">Read more</a>
+    </section>
+    <section class="about-us">
+       <h2>About us</h2>
+       <div class="about-us-text">
+        <p class="text">Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla partiatur. Excepteur sint occae cat cupidatat non proident, sunt in culpa qui officia dese runt mollit anim id est labourm velit esse cillum odlore eu fugiat nulla pariatu epreur sint occaecat</p>
+       </div>
+      <div class="high-quality">
+        <img src="" alt="" class="number01">
```

Comment:
@Alinchik1 you can delete this numbers, we will do it via css, no img tags needed

Reply:
deleted